poutyne/framework/experiment.py

```diff
@@ -253,11 +394,49 @@ def train(self,
               batches_per_step=1,
               seed=42):
         # pylint: disable=too-many-locals
-        if seed is not None:
-            # Make training deterministic.
-            random.seed(seed)
-            np.random.seed(seed)
-            torch.manual_seed(seed)
+        """
+        Trains or finetunes the attribute model on a dataset using a loader. If a previous training already occured
+        and lasted a total of `n_previous` epochs, then the model's weights will be set to the best previous checkpoint
+        and the training will be resumed for epochs range (`n_previous`, `epochs`].
+
+        Args:
+            train_loader: Generator-like object for the training set. See :func:`~Model.fit_generator()`
+                for details on the types of generators supported.
+            valid_loader (optional): Generator-like object for the validation set. See
+                :func:`~Model.fit_generator()` for details on the types of generators supported.
+                (Default value = None)
+            callbacks (List[~poutyne.framework.callbacks.Callback]): List of callbacks that will be called during
+                training.
+                (Default value = None)
+            lr_schedulers (List[~poutyne.framework.callbacks.lr_scheduler.LRScheduler]): List of learning rate
+                schedulers.
+                (Default value = None)
+            save_every_epoch (bool, optional): Whether or not to save the attribute model's weights after
```

Comment:
What do you mean by "attribute model"?

Reply:
The model that is accessed here via self.model, do not know how to rename but I am open to suggestions.

Reply:
Maybe just model?

Reply:
̀the experiment's model`

Reply:
```suggestion
            save_every_epoch (bool, optional): Whether or not to save the experiment model's weights after
```